Config/Inc/config.h

```diff
@@ -33,6 +33,8 @@ typedef enum SafetyStatus_e {SAFE = 0, DANGER = 1, OVERVOLTAGE = 2, UNDERVOLTAGE
 #define NUM_BATTERY_MODULES				31		// Number of battery modules
 #define NUM_TEMPERATURE_SENSORS			62		// Number of temperature sensors
 #define NUM_TEMP_SENSORS_PER_MOD		2		// Number of temperature sensors per battery module
+#define NUM_BATTERY_MODULES_MISSING     1       // Number of battery modules less than maximum value per minion board
```

Comment:
I think it would be better to define this in terms of `MAX_VOLT_SENSORS_PER_MINION_BOARD` (it might make sense to rename this to something like `NUM_BATTERY_MODULES_PER_MINION`) and `NUM_BATTERY_MODULES`, so we don't have to change the config file in as many places if we change the pack configuration